src/postprocessor.rs

```diff
@@ -244,6 +246,25 @@ impl Postprocessor for LengthPrefix {
     }
 }
 
+#[derive(Clone, Default)]
+pub(crate) struct TextualLength {}
+impl Postprocessor for TextualLength {
+    #[cfg(not(tarpaulin_include))]
+    fn name(&self) -> &str {
+        "textual-length"
```

Comment:
What do you think about this, to be consistent with the lookup key above?

```suggestion
        "textual-length-prefix"
```